mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -182,9 +182,24 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7311 openEditSignScreen (Lnet/minecraft/class_2625;)V
 		ARG 1 sign
 	METHOD method_7313 updateCapeAngles ()V
-	METHOD method_7315 openEditBookScreen (Lnet/minecraft/class_1799;Lnet/minecraft/class_1268;)V
+	METHOD method_7315 useBook (Lnet/minecraft/class_1799;Lnet/minecraft/class_1268;)V
+		COMMENT Called when the player uses (defaults to right click) a writable or written
+		COMMENT book item.
+		COMMENT
+		COMMENT <p>This can be called either on the client or the server player. Check {@code
+		COMMENT item} for whether this is a written or a writable book.
+		COMMENT
+		COMMENT @implNote The writing of a writable book in vanilla is totally controlled by
+		COMMENT the client; the server cannot make the client open a book edit screen by
+		COMMENT making a server player use a writable book. Only when the client finishes
+		COMMENT writing a book will it send a book update C2S packet.
```

Comment:
```suggestion
		COMMENT writing a book it will send a {@linkplain net.minecraft.network.packet.c2s.play.BookUpdateC2SPacket book update C2S packet}.
```